src/infra/execute.sh

```diff
@@ -14,4 +14,4 @@ kubectl apply -f app-configmap.yaml
 kubectl apply -f app-service.yaml
 kubectl apply -f app-deployment.yaml
 
-#minikube service user-service
\ No newline at end of file
+minikube service user-service
```

Comment:
since you are using `minikube` here, it's better to name script `local.sh` or `minikube.sh`